masterlist.yaml

```diff
@@ -14078,7 +14085,10 @@ plugins:
         util: 'SSEEdit v4.0.3'
 
   - name: 'Skyshards.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/6331/' ]
+    req: [ 'Unofficial Skyrim Special Edition Patch.esp' ]
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/1939-eso-skyshards/'
+	  - 'https://bethesda.net/en/mods/skyrim/mod-detail/3326552/'
```

Comment:
This url needs to be indented the same length than the url above, so two empty spaces compared to `url` are needed.